File: app/src/main/java/com/example/place4u/MainActivity.java
package com.example.place4u;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import android.util.Log;
import org.opencv.android.OpenCVLoader;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (!OpenCVLoader.initDebug()) {
            Log.e("test", "OpenCV 초기화 실패!");
        } else {
            Log.d("test", "OpenCV 초기화 성공!!!!!");
        }
    }
}

File: app/src/main/java/com/kaist/dd/MainActivity.kt
package com.kaist.dd

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.kaist.dd.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.startButton.setOnClickListener {
            var intent: Intent = Intent(this, DetectActivity::class.java)
            startActivity(intent)
        }
    }
}